### DISCUSSION TITLE
Ablauf wenn man im Ausland aktiv ist ?

### DISCUSSION THREAD

**Discussion by ds2k5** [wavelog/wavelog#454] [Q&A] [answered]
Hallo Zusammen,
wie geht man vor wenn man im Ausland aktiv ist ?
Sprich was muss man im Setup machen 

https://.../wavelog/index.php/stationsetup

Danke

**Comment by HB9HIL** [comment, marked as answer]
Hallo @ds2k5 

hierfür legst du eine neue Station Location an und verlinkst diese mit einem Logbuch. Du kannst hierfür ein neues Logbuch anlegen, um die QSO's von den "Haupt-Locations" besser zu trennen oder aber dein bisheriges Logbuch verwenden. 

In der Location gibst du als Station-Callsign einfach dein Rufzeichen mit dem entsprechenden Prefix an, z.B. DL/HB9HIL und alle anderen Stations-revelanten Daten. Bedenke dabei, dass korrekte DXCC auszuwählen (natürlich nicht dein Heimat-DXCC).

**Comment by ds2k5** [comment]
@HB9HIL 

danke
